.github/workflows/publish.yml

```diff
@@ -0,0 +1,41 @@
+name: docker package publish
+
+on:
+  push:
+    tags:
+      "*.*"
```

Comment:
Will it match `1.0.1`? Is it a regular expression or something else here?

Reply:
Yes.
All will be matched:
1.2
1.2.3
1.2.3.4
1.2.3.dev